<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleStyleWidget</class>
 <widget class="QWidget" name="SimpleStyleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>281</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QToolButton" name="alignLeft">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="alignCenter">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="alignRight">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="alignBlock">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4" rowspan="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="5">
      <widget class="QToolButton" name="superscript">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="6" rowspan="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="7">
      <widget class="QToolButton" name="decreaseIndent">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="bold">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="italic">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="underline">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="strikeOut">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QToolButton" name="subscript">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QToolButton" name="increaseIndent">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fontsFrame"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="textColor">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="backgroundColor">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="reversedText"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>List Style:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="listType">
       <property name="maxVisibleItems">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="listStyleAgain">
       <property name="toolTip">
        <string>Apply again</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>alignLeft</tabstop>
  <tabstop>alignCenter</tabstop>
  <tabstop>alignRight</tabstop>
  <tabstop>alignBlock</tabstop>
  <tabstop>bold</tabstop>
  <tabstop>italic</tabstop>
  <tabstop>underline</tabstop>
  <tabstop>strikeOut</tabstop>
  <tabstop>superscript</tabstop>
  <tabstop>subscript</tabstop>
  <tabstop>decreaseIndent</tabstop>
  <tabstop>increaseIndent</tabstop>
  <tabstop>textColor</tabstop>
  <tabstop>backgroundColor</tabstop>
  <tabstop>reversedText</tabstop>
  <tabstop>listType</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
